### PR TITLE
Show password-badge for nzbs with passwords (#274)

### PR DESCRIPTION
**Pull request by guaguasi**
Label in blue denoting "encrypted" file with password displayed via title attribute on mouse over.

### CODE REVIEW

File: webui/downloads.js [hunk 1]
```diff
@@ -769,7 +770,22 @@ var DownloadsUI = (new function($)
...
+		for (var i=0; i < parameters.length; i++) {
+			if (parameters[i]['Name'] === '*Unpack:Password' &&
+				parameters[i]['Value'])
+			{
+				isEncrypted = parameters[i]['Value'];
+				break;
+			}
+		}
+		return isEncrypted !== false ?
+			' <span class="label label-info" title="'+ isEncrypted +'">ENCRYPTED</span>' : '';
```
Comment (hugbug) [inline webui/downloads.js]: Use `Util.textToAttr(isEncrypted)`.
Also please use lower case for "ENCRYPTED" like for all other badges.

File: webui/downloads.js [hunk 2]
```diff
@@ -769,7 +770,22 @@ var DownloadsUI = (new function($)
...
-	
+
+	this.buildEncryptedLabel = function(parameters) {
+		var isEncrypted = false;
+
+		for (var i=0; i < parameters.length; i++) {
+			if (parameters[i]['Name'] === '*Unpack:Password' &&
+				parameters[i]['Value'])
+			{
+				isEncrypted = parameters[i]['Value'];
```
Comment (hugbug) [inline webui/downloads.js]: I don't understand why you assign string variable to boolean?

File: webui/downloads.js [hunk 3]
```diff
@@ -769,7 +770,22 @@ var DownloadsUI = (new function($)
...
+				break;
+			}
+		}
+		return isEncrypted !== false ?
```
Comment (hugbug) [inline webui/downloads.js]: Here isEncrypted can be either boolean (false) or a string. Is it ok to compare string with `false`?

File: webui/downloads.js [hunk 4]
```diff
@@ -769,7 +770,22 @@ var DownloadsUI = (new function($)
 			return ' <span class="label label-priority label-info">priority: ' + priority + '</span>';
 		}
 	}
-	
+
+	this.buildEncryptedLabel = function(parameters) {
```
Comment (hugbug) [inline webui/downloads.js]: please don't use dangling braces `{` for format consistency.

**Comment by hugbug** [comment]
Nice done BTW 👍
I hope the comments don't sound too harsh.

**Comment by guaguasi** [comment]
Not at all. lol.

I had originally started not having a label title but changed it midway and just never renamed the variable. You're right about the `false` comparison. `encryptedPassword` is only ever going to be undefined, a blank string, or an actual password. My original intent was to account for other falsy values but then I realized I'm dumb.

**Pull request merged**